src/pages/FilesPage/FilesPage.tsx

```diff
@@ -131,6 +133,15 @@ function FilesPage() {
 		setCheckedItems([]);
 	}, [currentLocation]);
 
+	useSocket([
+		{
+			type: 'FilesListUpdated',
```

Comment:
I think it's better to put the type in a constant in a separate file, where you can store other constants for web sockets